article/sections/_7_latent_confounding.tex

```diff
@@ -0,0 +1,209 @@
+
+
+
+\subsection{Case study: Simulation}
+\label{sec:deconfounder-simulation}
+
+The purpose of this section is to investigate the effectiveness of the 
+deconfounder algorithm \citep{wang_2019_blessings} in adjusting for unobserved 
+confounding. We use a simulated mode choice data where travel distance 
+linearly confounds both travel time and travel cost. We then mask the travel 
+distance data and treat it as an unobserved variable. 
+
+
+We estimate three models:
+
+\begin{itemize}
+	\item Model 1: A multinomial logit with the correct original specification, 
+	EXCEPT we ommit the travel distance variable in the specification without 
+	trying to adjust for it. This model represents the worst case scenario 
+	where a modeler ignores, or is unaware of, unobserved confounding.
+	\item Model 2: We use the deconfounder algorithm to try to recover the 
+	confounder (travel distance). In this method, we use all the variables in 
+	each mode's utility to recover that mode's confounder. This is in line 
+	with the approach taken in \citet{wang_2019_blessings}, where they use all the set 
+	of observed variables in the factor model to recover a substitute 
+	confounder.
+	\item Model 3: We use the deconfounder algorithm to try to recover the 
+	confounder (travel distance), but this time, we only use travel time and 
+	cost in the factor model, instead of all the variables in the utility 
+	specification of each mode. By only using what we know are confounded 
+	variables to recover the substitute confounder, our goal is to analyze 
+	whether we can improve the accuracy of this approach by adopting a more 
+	thoughtful approach to hypothesizing on the confounder variables. This can 
+	be in the form of building and testing candidate causal graphs that 
+	try to illustrate this confounding. 
+\end{itemize}
+
+
+We compare both the estimates of the coefficients on travel time and cost from 
+each of those three models to the true estimates used in the simulation, as well as the distribution of the recovered substitute confounder under each of 
+models 2 and 3 to the true confounder. The main findings of this exercise are the following:
+
+
+Using the true variables believed to be confounded (i.e. method 3 where only 
+travel time and cost are used to recover the confounder) leads to a better 
+recovery of the true confounder. Figure \ref{fig:qq-plot-method-2} and \ref{fig:qq-plot-method-3} show a QQ plot of the true 
+and recovered confounders under models 2 and 3 respectively. Looking at those 
+figures, one can see that the distribution of the recovered substitute 
+confounder under method 3 is closer to that of the true confounder than method 
+2. This suggests that it may be better to run the deconfounder algorithm based 
+on a hypothesized causal graph, rather than just running it on all the 
+observed covariates. Please refer to Section \ref{sec:graph-construction} for how to build plausible causal graphs. 
```

Comment:
I'm a little confused by this. Do you mean it might be better to run it on covariates of interest given a certain causal graph?